### PR TITLE
I made a button, Input.....

### PR DESCRIPTION
**Pull request by Moser2010**
Well I made what I could. No clue how to do anything else. sigh Direction would be fantastic.

### CODE REVIEW

**Comment by barberboy** [comment]
Let's refresh what you currently have:
1. You have a `<input>` element
2. You have a `<button>` element that says "Cody"
3. When you click on the `button` element, it adds the number `2` to the input

Since you know how to create a button, the next step is to create buttons for the numbers 0–9 and `+`, `-`, `*` `/`, and `=`.

When the buttons are clicked, instead of adding the number `2` to the input's `value` property, use `event.target.textContent` to get the button text, and add it instead.

When the equals button is clicked, pass the input's `value` to the `math.eval()` function, and update the input value with the result.

Then style with CSS and you're done.

**Comment by barberboy** [comment]
This is good progress, Cody! Keep at it.

**Comment by barberboy** [comment]
A bit spartan in style, but it is functional—good work!

File: calculator.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+function buttons() {
+    for (var i = 0; i < array.length; i++) {
+        makeButton(array[i]);
+
+        console.log(i);
+    }
+}
+
+makeInput();
+br();
```
Comment (barberboy) [inline calculator.js]: Just calling `br()` has no effect. `br()` creates and returns a new `br` element, but you don't actually use it or append it to the page.

File: calculator.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    button.addEventListener('click', function() {
+        buttonNum(button,document.getElementById('DtaInput'));
+        
+    }, false);
+    document.body.appendChild(button);
+    return button;
+}
+//if number inside of button is == 1 return "its 1"
+
+function buttonNum(button,display) {
```
Comment (barberboy) [inline calculator.js:39]: Why is this called `buttonNum`?

File: calculator.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
+//array of numbers for our buttons
+var array = [1, 2, 3, 4, 5, 6, 7, 8, 9, "+", "-", "*"]
+
+//makes the Input
+
+function makeInput() {
+    var mkInput = document.createElement("input");
+    mkInput.className = "cal-input";
+    mkInput.id = "DtaInput";
```
Comment (barberboy) [inline calculator.js:11]: I strongly recommend keeping `id` attributes all lower case, dash separated if necessary.

That being said, you don't even need an `id` attribute. You could grab the calculator input by calling `document.querySelector('.cal-input')`.

File: calculator.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    document.createElement('br');
+    return br
+}
+//this is not used
+//var num = event.target.textContent;
+
+
+//this puts text inside of our input when we do our click event.
+function handleButton(button) {
+    debugger;
```
Comment (barberboy) [inline calculator.js]: Please either comment out or remove debugger statements when you submit the assignment.

File: calculator.js [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+function br() {
+    document.createElement('br');
+    return br
+}
+//this is not used
+//var num = event.target.textContent;
+
+
+//this puts text inside of our input when we do our click event.
+function handleButton(button) {
```
Comment (barberboy) [inline calculator.js:18]: You never actually use this function. You could remove it entirely.

**Comment by barberboy** [comment]
@Moser2010 I left some comments for additional things to change before closing this, and you still need to style the calculator. Let me know if you have any questions.

**Comment by Moser2010** [comment]
@barberboy I styled the page.

**Comment by barberboy** [comment]
I just noticed you're missing buttons for `/` and `0`.

**Comment by barberboy** [comment]
:+1:  :100:

**Pull request closed without merging**